Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -625,7 +629,7 @@ function QuestieOptions.tabs.general:Initialize()
                 type = "toggle",
                 order = 2.11,
                 name = function() return l10n('Quest completed'); end,
-                desc = function() return l10n('Play a short sound when completing a quest so it is ready to turn in.'); end,
+                desc = function() return l10n('Play a short sound when completing a quest when it is ready to turn in.'); end,
```

Comment:
Good catch, whoops 😅 Remember to fix the key as well:

https://github.com/Questie/Questie/blob/35d09da8f998e74ea47f88dfd86ec2a059a918a4/Localization/Translations/Options/General.lua#L1385